ubuntu24.04/Dockerfile

```diff
@@ -64,8 +64,12 @@ RUN echo "TARGETARCH=$TARGETARCH"
 ADD install.sh /tmp
 
 # Fetch GPG keys for CUDA repo
-RUN apt-key del 7fa2af80 && OS_ARCH=${TARGETARCH/amd64/x86_64} && OS_ARCH=${OS_ARCH/arm64/sbsa} && \
-    apt-key adv --fetch-keys "https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2404/${OS_ARCH}/3bf863cc.pub"
+RUN apt-key del 3bf863cc && \
```

Comment:
Looks like this statement is running in the final layer, can we `apt purge` the curl package after performing this action?

Reply:
Let's also add this line as it helps shrink the image size further

```
rm -rf /var/lib/apt/lists/*; \
```